### PR TITLE
Reorganize ROSA Classic workflows

### PR DESCRIPTION
**Pull request by yasun1**
[OCPQE-20551](https://issues.redhat.com//browse/OCPQE-20551)
[OCPQE-20588](https://issues.redhat.com//browse/OCPQE-20588)

### CODE REVIEW

**Comment by yasun1** [comment]
/pj-rehearse periodic-ci-openshift-openshift-tests-private-master-rosacli-aws-rosa-security-group-e2e-f3 periodic-ci-openshift-openshift-tests-private-master-rosacli-aws-rosa-localzone-e2e-f3 periodic-ci-openshift-openshift-tests-private-master-rosacli-aws-rosa-private-proxy-e2e-f7 periodic-ci-openshift-openshift-tests-private-master-rosacli-aws-rosa-byo-kms-oidc-e2e-f3

**Comment by yasun1** [comment]
/pj-rehearse periodic-ci-openshift-openshift-tests-private-master-rosacli-aws-rosa-security-group-e2e-f3 periodic-ci-openshift-openshift-tests-private-master-rosacli-aws-rosa-localzone-e2e-f3 periodic-ci-openshift-openshift-tests-private-master-rosacli-aws-rosa-private-proxy-e2e-f7 periodic-ci-openshift-openshift-tests-private-master-rosacli-aws-rosa-byo-kms-oidc-e2e-f3

**Comment by yasun1** [comment]
/pj-rehearse periodic-ci-openshift-openshift-tests-private-master-rosacli-aws-rosa-non-sts-byo-vpc-kms-proxy-e2e-f3

**Comment by yasun1** [comment]
@liangxia Please help review. Thanks

File: ci-operator/jobs/openshift/openshift-tests-private/openshift-openshift-tests-private-master-periodics.yaml [hunk 1]
```diff
@@ -1387,18 +1583,7 @@ periodics:
     ci-operator.openshift.io/variant: rosacli
     ci.openshift.io/generator: prowgen
     pj-rehearse.openshift.io/can-be-rehearsed: "true"
-  name: periodic-ci-openshift-openshift-tests-private-master-rosacli-aws-rosa-sts-upgrade-e2e-f7
-  reporter_config:
-    slack:
-      channel: '#ocm-qe-prow-ci-jobs'
```
Comment (liangxia) [inline ci-operator/jobs/openshift/openshift-tests-private/openshift-openshift-tests-private-master-periodics.yaml:1393]: Remove slack notification here is expected ?

Comment (yasun1) [reply ci-operator/jobs/openshift/openshift-tests-private/openshift-openshift-tests-private-master-periodics.yaml:1393]: Missed this. Let me add it back. Thanks

**Comment by yasun1** [comment]
@liangxia Please help review again.

**Pull request merged**